docs-new/docs/index.md

```diff
@@ -1,3 +1,8 @@
 # Keptn Documentation
 
 This is the documentation homepage.
+If you are new to Keptn, please check out the following sections:
+
+- [Core Concepts](./core-concepts/index.md) will help you understand what Keptn is all about
+- [Get Started](./getting-started/index.md) will help you get going with your first use cases for Keptn
+- [Installation](./installation/index.md) will guide you through the installation procedure of Keptn
```

Comment:
> I specifically only put in the sections that are important for newcomers here. I think that makes more sense since the other sections can be found easily and they are not for new people. WDYT?

So where is the bridge for people who stick around after they are newcomers if it isn't here?  I suppose one option is to break it into two lists.  Lead into the first list with "The following sections give you the information you need to get started:" and then start a new list at line 12 with a lead-in like "The following sections provide additional information to help you as you implement keptn:" or something.

Thinking about the "user journey"...  I will fix the getting started guides so they link to the other docs more but, other than that, once I install and enable Keptn, I don't have anything useful running, do I?  So a noobie needs the rest of the docs fairly quickly.

But that's just my opinion -- it's your call.  But yes, let's get this merged -- when I saw the new site in its current state, I cringed a bit...  ;-)

Reply:
ah i'd prefer having one list instead of 2 then. i see your point but then why not just have one TOC here that has all of the information and users can decide for themselves with the corresponding descriptions of the content